compiler/rustc_const_eval/src/const_eval/error.rs

```diff
@@ -139,10 +139,9 @@ where
             ErrorHandled::TooGeneric(span)
         }
         err_inval!(AlreadyReported(guar)) => ErrorHandled::Reported(guar, span),
-        err_inval!(Layout(LayoutError::ReferencesError(guar))) => ErrorHandled::Reported(
-            ReportedErrorInfo::tainted_by_errors(guar),
-            span,
-        ),
+        err_inval!(Layout(LayoutError::ReferencesError(guar))) => {
+            ErrorHandled::Reported(ReportedErrorInfo::tainted_by_errors(guar), span)
+        }
```

Comment:
Surprisingly, this exact line was changed very recently in https://github.com/rust-lang/rust/pull/124516 - shouldn't tidy have fired on that?

Reply:
One would think so. I've been looking a bit at the code that `x fmt` uses to decide which files to format. It's surprisingly complex, and I wouldn't be surprised if it has bugs. I have also seen multiple cases where `x fmt` doesn't format a file that it should, and then I run it again and it does format it, though I can't determine any pattern to this. So in general I have suspicions that `x fmt` isn't entirely reliable in that way, and it's possible that tidy has similar issues.